src/BLDC_sm.c

```diff
@@ -312,27 +312,27 @@ void BL_State_Ctrl(void)
 
     if( BL_ARMING == bl_opstate )
     {
-      const uint16_t ARMING_TIME_100 = 0x08FF;
+      static const uint16_t ARMING_TIME_100 = 0x08FFu;
```

Comment:
Probably be good to setup constants for the other magik numbers e.g. 
      static const uint16_t ARMING_TIME_TOTAL = 0x0900; // 0x08FFu;
      static const uint16_t ARMING_TIME_DELAY = 0x0200u;
      static const uint16_t ARMING_TIME_MASK = 0x01C0u;
      static const uint16_t ARMING_BL_TIMING = 0x0010u;

Reply:
Something like that ... can't recall where the _100 came from so a better naming scheme might be in order! I think the number is counts * 1mS.